test/stepDefinitions/caseDetailsSteps.js

```diff
@@ -123,4 +123,24 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
     await caseDetailsPage.startEvent(event);
   });
 
+  When(/^I click the document link$/, async function () {
+    await caseDetailsPage.clickDocumentLink();
+  });
+
+  When(/^the media viewer is opened in a new tab$/, async function () {
+    let newTabUrl = await caseDetailsPage.getMediaViewerURL();
+    expect(newTabUrl.includes('/media-viewer')).to.true;
+  });
+
+  Then(/^the pdf document is visible in the new tab$/, async function () {
+    await caseDetailsPage.pdfContentVisible();
```

Comment:
Should this be
let visible = await caseDetailsPage.pdfContentVisible();
expect(visible).to.be.true;
?

Reply:
Of course. Thank you.